internal/handlers/pg/msg_update.go

```diff
@@ -68,6 +68,7 @@ func (h *Handler) MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 	}
 
 	var created bool
+
```

Comment:
All changes in this file are whitespace changes that also make this file different from Tigris version. Why we need them?

Reply:
Restored as it was. `MsgUpdate` and `MsgFindAndModify` are a huge mess. I think we should refactor them.